src/PaymentBundle/DependencyInjection/Configuration.php

```diff
@@ -35,6 +35,7 @@ public function getConfigTreeBuilder()
                 ->scalarNode('selector')->defaultValue('sonata.payment.selector.simple')->cannotBeEmpty()->end()
                 ->scalarNode('generator')->defaultValue('sonata.payment.generator.mysql')->cannotBeEmpty()->end()
                 ->arrayNode('transformers')
+                    ->addDefaultsIfNotSet()
```

Comment:
You asked to add a config test for PaymentBundle. And this is a config bug.
So there are two options:
1) Fix it in this PR
2) Add incomplete test, create another PR with this fix and test fix